docs/content/en/docs/user-guide/configuration-reference.md

```diff
@@ -181,6 +181,7 @@ spec:
 | Field | Type | Description | Required |
 |-|-|-|-|
 | name | string | The event name. | Yes |
+| createPullRequest | bool | Whether to create a new branch or not when commit changes in event watcher. | No |
```

Comment:
Same as above, only docs-dev should be update since this feature has not yet released.